projects/templedao/index.js

```diff
@@ -0,0 +1,33 @@
+const { staking } = require("../helper/staking");
+const { sumTokensAndLPsSharedOwners } = require("../helper/unwrapLPs");
+
+const templeStakingContract = "0x4D14b24EDb751221B3Ff08BBB8bd91D4b1c8bc77";
+const TEMPLE = "0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7";
+
+const templeTreasuryContract = "0x22c2fE05f55F81Bf32310acD9a7C51c4d7b4e443";
+const FRAX = "0x853d955acef822db058eb8505911ed77f175b99e";
+
+async function treasuryTvl() {
+  const balances = {};
+
+  await sumTokensAndLPsSharedOwners(
+    balances,
+    [
+      [FRAX, false],
+      [TEMPLE, false],
```

Comment:
temple in the treasury doesn't seem to be temple placed there by users but instead seems to be coins that are yet unissued, in that case i don't think we should count these as TVL (these coins for all intents and purposes don't exist yet since they are not circulating, in similar situations, like uniswap's treasury, we don't count these as TVL)

Reply:
fixed this

Reply:
I undersand, this one seems like olympus dao, asgard dao, etc.. I thougth we should count it in the same way, otherwise we just have staking part, de we ignore the treasury part then?